### PR TITLE
:memo: Gitmoji standard

### LINKED ISSUE #390: start using gitmojis
[As seen here.](https://gitmoji.dev/) It might be useful to stick to an emoji standard on the commit messages and PRs. Carlos Cuesta explains [here](https://gitmoji.dev/about).

### PR DESCRIPTION
**Pull request by EldaveO**
##### Summary 

:memo: Addition of the gitmoji standard and creation of the contributing guidelines.
Reviewed using a markdown editor.
resolves #390 

##### Checklist

* [ ] this is a source code change
  * [ ] run `isort . && black .` in the repository root
  * [ ] run `pytest` in the repository root
  * [x] link to issue being fixed using a [_fixes keyword_](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue), if such an issue exists
  * [ ] update the wiki documentation if necessary
* [x] or, this is **not** a source code change

### CODE REVIEW

File: .github/contributing_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#### Summary
```
Comment (twentylemon) [inline .github/contributing_guidelines.md]: This file does not have the correct name. See [docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors).

File: .github/contributing_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#### Summary 
+
+Contributing to Duckbot can be a fun and rewarding experience! 
+Before you contribute, please review these guidelines so that consistancy can be maintained.
+
+##### Gitmoji standard
```
Comment (twentylemon) [inline .github/contributing_guidelines.md]: It's not really a _standard_, just something we're using.

**Review by twentylemon — CHANGES_REQUESTED** [review]
You didn't even use an emoji in this pull request. Come on dude.

File: .github/contributing_guidelines.md (hunk 2 above)
Comment (EldaveO) [reply .github/contributing_guidelines.md]: Noted, I've updated the wording in the lasted commit to move away from the mention of a defined _standard_.

File: .github/contributing_guidelines.md (hunk 1 above)
Comment (EldaveO) [reply .github/contributing_guidelines.md]: Fixed it in my latest commit.
Thank you!

**Review by twentylemon — CHANGES_REQUESTED** [review]
Thinking more about this, I don't think `CONTRIBUTING` is the best place for this. We don't really want contributors and likely won't refer or add to these guidelines for that reason. Adding to the existing pull request template is sufficient for fixing #390, and more likely to serve as a reminder to people opening pull requests to actually add the emojis -- hiding them in `CONTRIBUTING` will lead to a lot of forgotten emojis.

Also,

> You didn't even use an emoji in this pull request. Come on dude.

Actually though.

**Comment by Chippers255** [comment]
bruh
> You didn't even use an emoji in this pull request. Come on dude.

**Comment by EldaveO** [comment]
So for PRs the emojis should be in the tile rather than in the summary section.

Lesson learned.

**Comment by twentylemon** [comment]
@EldaveO attaching `needs reply` label. Get some traction or it's getting yeeted.

**Comment by jameshughes89** [comment]
> Thinking more about this, I don't think `CONTRIBUTING` is the best place for this. We don't really want contributors and likely won't refer or add to these guidelines for that reason. **Adding to the existing pull request template is sufficient** for fixing #390, and more likely to serve as a reminder to people opening pull requests to actually add the emojis -- hiding them in `CONTRIBUTING` will lead to a lot of forgotten emojis.

+1 for template.

**Comment by jameshughes89** [comment]
I'm still down, but for a simpler, lightweight and minimalist version.

**Comment by EldaveO** [comment]
Yeet away, I'll bring it back up when I get more time.

**Pull request closed without merging**